Cargo.toml

```diff
@@ -1,58 +1,8 @@
-[package]
-name = "iota-lib-rs"
-edition = '2018'
-version = "0.4.1"
-authors = ["Nathan Jaremko <nathan@jaremko.ca>", "Yu-Wei Wu <wusyong9104@gmail.com>"]
-license = "MIT"
-readme = "README.md"
-keywords = ["iota", "api"]
-repository = "https://github.com/iotaledger/iota.rs"
-homepage = "https://github.com/iotaledger/iota.rs"
-documentation = "https://docs.rs/iota-lib-rs"
-description = "A rust implementation of the IOTA Client API"
-
-[lib]
-name = "iota_lib_rs"
-
 [workspace]
 members = [
-    #"iota-pow",
-    #"iota-crypto",
-    #"iota-conversion",
-    #"iota-model",
-    #"iota-utils",
-    #"iota-constants",
-    #"iota-validation",
-    #"iota-signing",
-    "iota-client"
+    "iota-sdk",
```

Comment:
Since this is a workspace, I would like to make every crates be member of it. I would like to rename with `iota` but that name is already taken from crates.io . I also so want the name be easy to identify as main crate so sdk is the closest I came up with.

Reply:
How about iota-core?

Reply:
sounds good